tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -820,6 +820,11 @@ def _init_model_capacity(self):
         self._init_max_seq_len()
         self._init_max_num_tokens()
 
+    def _release_cuda_graphs(self):
+        self._cuda_graphs.clear()
```

Comment:
If the graphs inside _cuda_graphs are not used anymore, then we need to reset the graph accordingly to give the memory back to system like 
```suggestion
        for graph in self._cuda_graphs:
            graph.reset()
        torch.cuda.empty_cache()
        self._cuda_graphs.clear()
        
```

Reply:
Thanks, changed as suggestion. I thought graph could release resource when it's deleted.